src/Export/Adapters/OptionsAdapter.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FINDOLOGIC\Shopware6Common\Export\Adapters;
+
+use FINDOLOGIC\Export\Data\Attribute;
+use FINDOLOGIC\Shopware6Common\Export\Config\PluginConfig;
+use FINDOLOGIC\Shopware6Common\Traits\AdapterHelper;
+use Vin\ShopwareSdk\Data\Entity\Product\ProductEntity;
+
+class OptionsAdapter
+{
+    use AdapterHelper;
+
+    public function __construct(PluginConfig $pluginConfig)
+    {
+        $this->pluginConfig = $pluginConfig;
+    }
+
+    /**
+     * @return Attribute[]
+     */
+    public function getOptionAttributes(ProductEntity $product): array
```

Comment:
Every adapter should just have one public function with the name `adapt()`, please rename this function.

Maybe it would make sense to create an `AdapterInterface` with the `adapt()` function and add `implements AdapterInterface` to all the adapters.

(`ExportItemAdapter` is an exception)